src/components/Checkboxes.tsx

```diff
@@ -0,0 +1,296 @@
+import {
+  Fieldset,
+  FieldsetPropsWithoutChildren,
+  fieldsetWithoutChildrenPropTypes
+} from "./Fieldset/Fieldset";
+import PropTypes, { ValidationMap } from "prop-types";
+import React, { ChangeEvent } from "react";
+import { Hint, HintProps } from "./Hint";
+import { ErrorMessage, ErrorMessageProps } from "./ErrorMessage";
+import {
+  FormGroup,
+  FormGroupPropsWithoutChildren,
+  formGroupWithoutChildrenPropTypes
+} from "./FormGroup";
+import { Label, LabelProps } from "./Label";
+import { Attributes, DataAttributes } from "../helpers/Attributes";
+import classNames from "classnames";
+import "lbh-frontend/lbh/components/lbh-checkboxes/_checkboxes.scss";
+
+export interface Checkbox extends React.AriaAttributes, DataAttributes {
```

Comment:
I think this name will have a clash with the name of a boolean checkbox (which I think we want for post visit actions). Suggest `CheckboxesItem` instead?

Reply:
Sorry what is the boolean checkbox? Given that this is the lbh-frontend-react library I'm keen to keep this as Checkbox. If it causes problems for MaT/THC we can rename in there?

Reply:
I mean the case where we want to be able to use a checkbox to represent a boolean value, rather than a list of values.